test/strategy/gradual-rollout-random.test.js

```diff
@@ -36,3 +36,9 @@ test('gradual-rollout-random strategy should be disabled when percentage=0', t =
     let params = { percentage: '0', groupId: 'test' };
     t.false(strategy.isEnabled(params));
 });
+
+test('gradual-rollout-random strategy should be enabled when percentage=100', t => {
```

Comment:
This test is kind of poor. We should control random-ness in the test to make sure we always test the edge cases

Reply:
I find useful the test added by @zjy01 here. But it certainly is not going for the edge cases.

How do you guys usually test that? [proxyquire](https://github.com/thlorenz/proxyquire) is how I would go.

Reply:
I usually just add the possibility to inject a random-generator-function in the constructor which defaults to `Math.random`. In the test you just provide a function which produces the number you want to test.